### ISSUE TITLE
Smart suggestion of labels based on previous transactions with similar descriptions

### ISSUE THREAD

**Issue by Bone008** [Bone008/finance-tracker#38]
TODO: Where and how should it be triggered?

Nice would be on import, but manually on all selected items is probably needed, too. Maybe an extra button next to group/split/edit/etc. that opens a dialog with all suggestions.

**Comment by Bone008** [comment]
Maybe a "create filter rule" button would be nice, once #15 is complete.

**Comment by Bone008** [comment]
For labels, this could be triggered with a small delay when opening the "inline add label" text field and when opening the "edit transaction" dialog.

It would be nice to allow previewing the matched transaction(s) that the smart suggestion is based on WITHOUT having to navigate away from the current filter, but this is nice to have.
